applications.go

```diff
@@ -0,0 +1,95 @@
+package solus
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+)
+
+type ApplicationsService service
+
+type LoginLinkType string
+
+const (
+	LoginLinkNone   LoginLinkType = "none"
+	LoginLinkURL    LoginLinkType = "url"
+	LoginLinkJSCode LoginLinkType = "js_code"
```

Comment:
Sorry `LoginLinkTypeJSCode`. Main thing is to repeat a type name in the constant 'cause Golang doesn't have enum it's a common way to name like that.